src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -178,8 +176,8 @@ def parse_performance_features(log_lines, strategies, arguments):
       'max_len': 0,
       'manual_dict_size': 0,
       'merge_edge_coverage': 0,
-      'merge_new_files': 0,
-      'merge_new_features': 0,
+      'new_cov_edges': 0,
```

Comment:
Actually, defaulting to zero is problematic right?

Reply:
thought more on -1. we can't do -1 since fuzzer-stats shows stuff by "sum(t.new_cov_features) as new_cov_features", so it need some support for exclusions format. And once we support exclusions format, we can just exclude by corpus_subset strategy name.